examples/issue-1695-fix-example.yaml

```diff
@@ -0,0 +1,100 @@
+# Example configuration demonstrating the fix for Issue #1695
```

Comment:
What is the reason for this file ? I haven't seen any other issue creating these.